src/Libraries/CoreNodeModels/CoreNodeModels.csproj

```diff
@@ -26,17 +26,11 @@
       <!--Exclude copying the runtime dlls because we will handle it in the BinaryCompatibilityOps target -->
       <ExcludeAssets>runtime</ExcludeAssets>
     </PackageReference>
-	  <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
```

Comment:
why remove Newtonsoft from net48?
Maybe you can create a common ItemGroup for both net48 and net6.0 and put it there ?

Reply:
it's referenced below on line 40.